tests/logictest/requiresments.txt

```diff
@@ -0,0 +1,4 @@
+mysql-connector>=2.2.9
```

Comment:
We are using poetry to maintain python deps.

See https://github.com/datafuselabs/databend/blob/main/tests/pyproject.toml for more details.

Reply:
OK，need a commit to use poetry instead of pip